### DISCUSSION TITLE
Consider adding zero.one/fxl

### DISCUSSION THREAD

**Discussion by zhoumin79** [babashka/babashka#955] [Ideas]
Fxl is a nice library to work with excel files.
It attempts to do differently to docjure and excel-clj is to represent spreadsheets as an unordered collection of maps, instead of relying on tabular formats. This makes it easier to deal with independent, smaller components of the spreadsheet and simply apply concat to put together different components of a spreadsheet.

In my everyday work, I need to do with many sheets with relations in one excel. Fxl is a good tool to deal with them.

BB is a great tool to automating work. But now it doesn't support Fxl. Wish a pod to support fxl to work with excel data. Many thanks!

**Comment by borkdude** [comment]
@zhoumin79 I made a babashka-like too for Node.js now: https://github.com/borkdude/nbb
Perhaps you're able to leverage some Node.js library meanwhile using that.

**Reply by zhoumin79** [reply]
Thanks. Will try.

**Reply by borkdude** [reply]
@zhoumin79 I made an example here: https://github.com/borkdude/nbb/blob/main/examples/exceljs/example.cljs
